lib/presentation/widget/destination_picker/destination_picker_widget.dart

```diff
@@ -354,4 +354,14 @@ class _DestinationPickerWidgetState extends State<DestinationPickerWidget> {
       style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
     );
   }
+
+  bool _isDestinationAvailable(DestinationType destinationType) {
+    if (_destinationPickerArguments != null
+        && _destinationPickerArguments.availableDestinationTypes != null
```

Comment:
`_destinationPickerArguments.availableDestinationTypes != null` is redundated because of `availableDestinationTypes` always initialized from `DestinationPickerArguments`'s constructor.

Reply:
And you can also write the shorter code here, i.e:
```dart
return (_destinationPickerArguments != null
        && _destinationPickerArguments.availableDestinationTypes != null
        && _destinationPickerArguments.availableDestinationTypes.contains(destinationType)
    );
```